launch_testing/test/launch_testing/test_launch_test_runner_validation.py

```diff
@@ -24,7 +24,8 @@
 
 
 def make_test_run_for_dut(generate_test_description_function):
-    module = imp.new_module('test_module')
+    test_spec = importlib.util.spec_from_loader('test_module', loader=None)
+    module = importlib.util.module_from_spec(test_spec)
```

Comment:
This change has already been merged in #407 and should therefore not be part of this PR anymore.

Reply:
Didn't even notice this was still in there. I will remove this fix from the PR.